cpp/oneapi/dal/table/backend/csr_table_builder_impl.hpp

```diff
@@ -25,9 +25,15 @@ namespace oneapi::dal::backend {
 
 class csr_table_builder_impl : public detail::csr_table_builder_template<csr_table_builder_impl> {
 public:
+#ifdef ONEDAL_DATA_PARALLEL
+    csr_table_builder_impl() : dependencies_(nullptr) {
```

Comment:
What's the necessity of usage dependencies here? In comparison with homogen_table, there is no dependencies in homogen_table_builder